packages/react-charting/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -373,7 +373,9 @@ export interface ICartesianChartProps {
   /**
    * Callout customization props
    */
-  calloutProps?: Partial<ICalloutProps>;
+  calloutProps?: Partial<ICalloutProps> & {
```

Comment:
Dont add a message to callout. You can customize the entire callout content. See onRenderCalloutPerDataPoint property and example in AreaChart basic example.

Reply:
Done. But the Callout card is not looking exactly same. To Make it exactly same we need to copy style classes from Cartesian chart.

Reply:
Uploading the screenshot for Callout , because changes are only in react-examples so demo site is not updating

![image](https://github.com/microsoft/fluentui/assets/103660888/bfb21c02-d4fe-426c-a3a6-b3b3fc9e5e83)